src/Microsoft.VisualStudio.ProjectSystem.CSharp.VS/Microsoft.VisualStudio.ProjectSystem.CSharp.VS.csproj

```diff
@@ -61,6 +61,9 @@
   </ItemGroup>
   <ItemGroup>
     <EmbeddedResource Include="VSPackage.resx" />
+    <EmbeddedResource Include="VSPackage.*.resx">
```

Comment:
Will the old project system expand this out to individual entries? Try adding a file to the project with this change.